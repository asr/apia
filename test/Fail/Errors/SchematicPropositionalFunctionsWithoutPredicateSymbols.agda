------------------------------------------------------------------------------
-- Incompatible options
------------------------------------------------------------------------------

{-# OPTIONS --exact-split              #-}
{-# OPTIONS --no-sized-types           #-}
{-# OPTIONS --no-universe-polymorphism #-}
{-# OPTIONS --without-K                #-}

-- The @--schematic-propositional-functions@ and
-- @--without-predicate-symbols@ options are incompatible.

module SchematicPropositionalFunctionsWithoutPredicateSymbols where

postulate D : Set

postulate id : {P : D → Set}{x : D} → P x → P x
{-# ATP prove id #-}
