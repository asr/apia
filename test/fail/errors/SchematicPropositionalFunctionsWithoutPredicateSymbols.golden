ret > ExitFailure 1
err > apia: use the ‘--schematic-propositional-functions’ option for the translation of first-order logic universal quantified propositional functions
err >
